Assets/Scripts/Enemies/EnemyAttack.cs

```diff
@@ -6,9 +6,12 @@ public enum AimType
 {
     Player,
     Direction,
+    SingleCollision,
+    MultipleCollision
```

Comment:
Could we be more descriptive here? I'm not sure what "Single Collision" or "Multiple Collision" means. Adding a code comment would also suffice to describe what's going on.